odl/test/tomo/backends/astra_setup_test.py

```diff
@@ -168,12 +168,6 @@ def test_astra_projection_geometry():
     apart = odl.uniform_partition(0, 2 * np.pi, 5)
     dpart = odl.uniform_partition(-40, 40, 10)
 
-    # motion sampling grid, detector sampling grid but not RegularGrid
-    dpart_0 = odl.RectPartition(odl.IntervalProd(0, 0), odl.TensorGrid([0]))
-    geom_p2d = odl.tomo.Parallel2dGeometry(apart, dpart=dpart_0)
-    with pytest.raises(ValueError):
-        odl.tomo.astra_projection_geometry(geom_p2d)
```

Comment:
Too lazy :-)
Okay I'll do it. I had to get it out because gates close at 7, didn't want to be locked in.